test/Inputs/clang-importer-sdk/usr/include/bridged_typedef.h

```diff
@@ -0,0 +1,13 @@
+#ifndef BRIDGED_TYPEDEF_H
+#define BRIDGED_TYPEDEF_H
+
+@import Foundation;
+
+
+typedef NSString *NSMyAmazingStringAlias __attribute__((swift_bridged_typedef));
```

Comment:
Can you test a typedef that uses this typedef as well? If that typedef doesn't have the `swift_bridged_typedef` attribute, what behavior should it have?